app/models/submission_file.rb

```diff
@@ -88,6 +90,10 @@ def is_pdf?
     File.extname(filename).casecmp('.pdf') == 0
   end
 
+  def pynb?
```

Comment:
I'd prefer `is_pynb?` here to be consistent with the other methods.

Reply:
It is consistent with other methods but not with the rubocop checks. We should actually swap the rest of them over to this style or if not, then disable the cop. I'm happy with either.

Reply:
Ah interesting, you can disable the cop then.